paper/paper.md

```diff
@@ -19,22 +19,22 @@ bibliography: paper.bib
 
 # Summary
 
-**BenchmarkDataNLP.jl** is a package written in Julia Lang for generating synthetic text corpora that can be used to systematically benchmark and evaluate Natural Language Processing (NLP) models such as RNNs, LSTMs, and Large Language Models (LLMs). By enabling users to control core linguistic parameters such as the alphabet size (characters selected from the Unicode Hangul block, Korean Language), vocabulary size, grammatical expansion complexity, and semantic structures this library can help users test (as well as evaluate or debug) NLP models. Instead of exposing many of the key internal parameter choices to the user a minimal number are so that users do not have to focus on how to correctly configure the generation process. The key value for the user is the *complexity* integer value which controls the size of the alphabet, vocabulary and grammar expansions. This allows a range starting from 1 so that very simple text corpora can be generated. For example at complexity=1 there are 5 letters in the alphabet and 10 words used in the vocabulary with 2 grammar roles when a Context Free Grammar generator is select. Users can choose how many independent productions are desired which are each supplied as entries in a .jsonl file. The defaults provided in the documentation should suffice for most use cases. 
+**BenchmarkDataNLP.jl** is a package written in Julia Lang for generating synthetic text corpora that can be used to systematically benchmark and evaluate Natural Language Processing (NLP) models such as Recurrent Neural Networks (RNNs), Long Short-Term Memory (LSTMs), and Large Language Models (LLMs). By enabling users to control core linguistic parameters such as the alphabet size (characters selected from the Unicode Hangul block, Korean Language), vocabulary size, grammatical expansion complexity, and semantic structures this library can help users test, evaluate, and debug NLP models. Instead of exposing many parameters to the user, it is kept at minimum, so that users do not have to focus on how to correctly configure the generation process. The key parameter is the *complexity* (integer value), which controls the size of the alphabet, vocabulary, and grammar expansions. This parameter accepts an integer in the range 1 to **XXX**. For example, at `complexity = 1` (simplest value), there are 5 letters in the alphabet and 10 words used in the vocabulary with 2 grammar roles when a Context Free Grammar generator is select. Users can choose how many independent productions are desired which are each supplied as entries in a .jsonl file. The defaults provided in the documentation should suffice for most use cases. 
```

Comment:
I don't understand this sentence:

    Users can choose how many independent productions are desired which are each supplied as entries in a .jsonl file.

What is a 'production'? (I'm not an expert in NLP - if this is how you define something, that's fine)
Does the user need to supply these productions in a .json file?

Reply:
Also: please, fill in the 'XXX' I've marked for you.